src/Streamly/Internal/Control/ForkLifted.hs

```diff
@@ -32,11 +33,23 @@ doFork :: MonadRunInIO m
     -> RunInIO m
     -> (SomeException -> IO ())
     -> m ThreadId
-doFork action (RunInIO mrun) exHandler =
+doFork = doFork' False
+
+-- | Similar to 'doFork', but has a "bound" boolean parameter for specifying
```

Comment:
Double quote in haddocks has the meaning of pointing to a module I believe.

Reply:
Thanks; you’re right. Resolved in separate PR: https://github.com/composewell/streamly/pull/2406